aasemble/django/apps/buildsvc/tests_functional.py

```diff
@@ -211,6 +211,26 @@ def verify_login_page(self):
         else:
             return True
 
+    def test_mirror_set(self):
+        '''This test verifies the working of mirror set'''
+        self.create_login_session('brandon')
+        # test whether sources page opens after user logs in
+        self.selenium.get('%s%s' % (self.live_server_url, '/buildsvc/sources/'))
```

Comment:
Can you please use reverse() to find the right url?

For API tests, we use hard coded URL's, because they're part of the API spec. For the web UI, the URL's aren't part of the specification, so they shouldn't be hard coded.